dapr/proto/runtime/v1/dapr.proto

```diff
@@ -148,7 +148,10 @@ service Dapr {
   rpc ResumeWorkflowAlpha1 (WorkflowActivityRequest) returns (WorkflowActivityResponse) {}
 
   // Raise Event for a Workflow
-  rpc RaiseEventWorkflowAlpha1 (RaiseEventWorkflowRequest) returns (RaiseEventWorkflowResponse) {}
+  rpc RaiseEventWorkflowAlpha1 (RaiseEventWorkflowRequest) returns (WorkflowActivityResponse) {}
+
+  // Purge Workflow
+  rpc PurgeWorkflowAlpha1 (PurgeWorkflowRequest) returns (WorkflowActivityResponse) {}
```

Comment:
Standard practice for APIs that don't return anything is to use `google.protobuf.Empty`. I suggest we do the same here.

Reply:
Looks like this got missed?